#File generated by Pentaho Translator for package 'org.pentaho.di.trans.steps.addsequence' in locale 'zh_CN'


#Mon May 05 15:12:39 CST 2008
AddSequenceDialog.NoValidConnectionError.DialogMessage=\u8BF7\u9009\u62E9\u4E00\u4E2A\u6709\u6548\u7684\u6570\u636E\u5E93\u8FDE\u63A5\!
AddSequenceDialog.Log.GettingKeyInfo=\u83B7\u53D6\u5173\u952E\u5B57\u4FE1\u606F...
AddSequence.Log.WriteRow=\u5199\u8BB0\u5F55 \#
AddSequence.Log.ConnectedDB=\u8FDE\u63A5\u6570\u636E\u5E93...
AddSequence.Exception.CouldNotFindNextValueForSequence=Couldn''t find next value for sequence \: 
AddSequence.Log.ErrorInStep=Because of an error, this step can''t continue\: 
AddSequenceDialog.TargetSchema.Label=\u6A21\u5F0F\u540D\u79F0
AddSequenceDialog.UseDatabase.Label=\u4F7F\u7528DB\u6765\u83B7\u53D6sequence? 
AddSequenceMeta.ErrorMessage.UnableToConnectDB=I was unable to connect to the database to verify the status of the table.
AddSequenceDialog.UseCounter.Tooltip=\u52FE\u4E0A\u8FD9\u4E2A\u4EE5\u4FBF\u4F7F\u7528\u5185\u90E8Kettle\u8BA1\u6570\u5668\u6765\u8BA1\u7B97sequence\u503C.
AddSequence.Log.CouldNotConnectToDB=Step couldn''t connect to the database\: 
AddSequenceDialog.UseDatabaseGroup.Label=\u4F7F\u7528\u6570\u636E\u5E93\u6765\u751F\u6210\u5E8F\u5217
AddSequence.Log.TransformationCountersHashtableNotAllocated=Sorry, Transformation counters hash table not allocated\! (internal error)
AddSequenceDialog.UseDatabase.Tooltip=\u52FE\u4E0A\u8FD9\u4E2A\u4EE5\u4FBF\u4F7F\u7528\u6570\u636E\u5E93(\u53EF\u80FD\u662FOracle)\u6765\u83B7\u53D6sequence.
AddSequenceMeta.Exception.UnableToReadStepInfo=Unable to read step information from the repository for id_step\=
AddSequenceMeta.CheckResult.NoInputReceived.Title=No input received from other steps\!
AddSequenceDialog.StartAt.Label=\u8D77\u59CB\u503C 
AddSequenceDialog.UseCounter.Label=\u4F7F\u7528\u8BA1\u6570\u5668\u6765\u8BA1\u7B97sequence? 
AddSequence.Log.NeedToSelectSequence=You need to select to create a sequence either using a database or an internal counter.
AddSequenceDialog.Valuename.Label=\u503C\u7684\u540D\u79F0 
AddSequenceDialog.NoValidConnectionError.DialogTitle=\u9519\u8BEF
AddSequence.Exception.NoSpecifiedMethod=No method is specified in this step\!
AddSequence.Log.ReadRow=\u8BFB\u8BB0\u5F55 \#
AddSequenceDialog.UseCounterGroup.Label=\u4F7F\u7528\u8F6C\u6362\u8BA1\u6570\u5668\u6765\u751F\u6210\u5E8F\u5217
AddSequenceDialog.CounterName.Label=\u8BA1\u6570\u5668\u540D\u79F0(\u53EF\u9009)
AddSequenceDialog.IncrBy.Label=\u589E\u957F\u6839\u636E 
AddSequenceMeta.ErrorMessage.NoConnectionDefined=There is no connection defined in this step.
AddSequence.Log.CouldNotParseCounterValue=\u4E0D\u80FD\u89E3\u6790 {0} \u7684\u8BA1\u6570\u5668\u503C,\u539F\u503C [{1}] \u53D8\u6210 [{2}]. \u9519\u8BEF\: {3}.
AddSequenceDialog.Shell.Title=\u4ECE\u6570\u636E\u5E93\u8BFB\u53D6sequence\u503C
AddSequence.Exception.ErrorReadingSequence=Error reading next value of sequence [{0}] from database
AddSequenceMeta.CheckResult.SequenceCouldNotBeFound.Title=The sequence [{0}] couldn''t be found
AddSequenceDialog.Seqname.Label=Sequence\u540D\u79F0
AddSequence.Log.CountersWithDifferentCharacteristics=\u8BA1\u6570\u5668 [{0}] \u88AB\u5B9A\u4E49\u4E86\u591A\u6B21
AddSequenceDialog.StepName.Label=\u6B65\u9AA4\u540D\u79F0 
AddSequenceMeta.Exception.ErrorLoadingStepInfo=Error loading step info from XML
AddSequence.Log.LineNumber=linenr 
AddSequenceMeta.Exception.UnableToSaveStepInfo=Unable to save step info to the repository for id_step\=
AddSequenceMeta.CheckResult.SequenceExists.Title=Sequence\u5DF2\u7ECF\u5B58\u5728.
AddSequenceDialog.MaxVal.Label=\u6700\u5927\u503C 
AddSequenceMeta.CheckResult.StepIsReceving.Title=Step is receiving info from other steps.
AddSequenceMeta.CheckResult.UnableToConnectDB.Title=Unable to connect to database to verify sequence because of an error\: 
